### PR TITLE
Ria 3314 decided notification

### PR DESCRIPTION
**Pull request by jkumar-moj**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/RIA-3314

### Change description ###
Home Office appeal decided notification

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/iahomeofficeintegrationapi/domain/handlers/presubmit/AppealDecidedNotificationHandler.java [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+        final String appealDecision = asylumCase.read(IS_DECISION_ALLOWED, AppealDecision.class)
+            .orElseThrow(() -> new IllegalStateException("decision property must be set"))
+            .getValue().toUpperCase();
+
+        final AppealDecidedInstructMessage bundleInstructMessage =
+            appealDecidedInstructMessage()
+                .withConsumerReference(notificationsHelper.getConsumerReference(caseId))
+                .withHoReference(homeOfficeReferenceNumber)
+                .withMessageHeader(notificationsHelper.getMessageHeader())
+                .withMessageType(COURT_OUTCOME.toString())
```
Comment (Lee365) [inline src/main/java/uk/gov/hmcts/reform/iahomeofficeintegrationapi/domain/handlers/presubmit/AppealDecidedNotificationHandler.java:78]: Try to use COURT_OUTCOME.name() as name() method is final, while the toString() may be overwritten.

File: src/main/java/uk/gov/hmcts/reform/iahomeofficeintegrationapi/domain/handlers/presubmit/AppealDecidedNotificationHandler.java [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+                .withCourtOutcome(new CourtOutcome(CourtType.FIRST_TIER, Outcome.valueOf(appealDecision)))
+                .withNote(AppealDecidedNote.valueOf(appealDecision).toString())
+                .build();
+
+        log.info("Finished constructing {} notification request for caseId: {}, HomeOffice reference: {}",
+            COURT_OUTCOME.toString(), caseId, homeOfficeReferenceNumber);
```
Comment (Lee365) [inline src/main/java/uk/gov/hmcts/reform/iahomeofficeintegrationapi/domain/handlers/presubmit/AppealDecidedNotificationHandler.java:84]: COURT_OUTCOME.name() is better

File: src/main/java/uk/gov/hmcts/reform/iahomeofficeintegrationapi/domain/handlers/presubmit/AppealDecidedNotificationHandler.java [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+
+        final String notificationStatus = homeOfficeInstructService.sendNotification(bundleInstructMessage);
+
+        asylumCase.write(AsylumCaseDefinition.HOME_OFFICE_APPEAL_DECIDED_INSTRUCT_STATUS, notificationStatus);
+
+        log.info("SENT: {} notification for caseId: {}, HomeOffice reference: {}, status: {}, event: {}",
+            COURT_OUTCOME.toString(), caseId, homeOfficeReferenceNumber, notificationStatus, callback.getEvent());
```
Comment (Lee365) [inline src/main/java/uk/gov/hmcts/reform/iahomeofficeintegrationapi/domain/handlers/presubmit/AppealDecidedNotificationHandler.java:91]: COURT_OUTCOME.name() is better

**Review by Lee365 — APPROVED** [review]
LGTM

**Review by Srihari-Nakkala — APPROVED** [review]
LGTM

**Comment by jkumar-moj** [comment]
Reopening the PR as it is closed by stale bot

**Pull request merged**